### PR TITLE
v15 Upgrade Guide

### PR DESCRIPTION
**Pull request by ferschubert-hm**
Upgrade guide for [v15 Beta](https://github.com/humanmade/product-dev/issues/1342)

### CODE REVIEW

File: other-docs/guides/upgrading/v15.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+For the environments still running in PHP 7.4 we have released new PHP images which receives backport of security fixes from maintained branches. 
+
+
+### Altis Core improvements 
+
+A number of modules and libraries have been updated to incorporate important bug fixes and improvements.
+
+### Documentation 
+
+Some of our developer focussed documentation hase been clarified and improved, taking on board feedback from our customers and partners.
```
Comment (wisyhambolu) [inline other-docs/guides/upgrading/v15.md:77]: change `hase` to `has`

File: other-docs/guides/upgrading/v15.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+- Site Editor out of beta
+- There is a new server-side [HTML parsing API](https://make.wordpress.org/core/2023/03/07/introducing-the-html-api-in-wordpress-6-2/)
+- Better PHP 8.0-8.2 compatibility with the updated Requests library
+- Bundled fonts in Themes
+
+### PHP 7.4 Security Support ###
+
+As one of our top priorities, security is taken seriously in Altis. In Altis v13 the support for PHP 7.4 has been removed, and customers asked to follow the [PHP Version Guide](docs://guides/updating-php/) for information on how to upgrade to PHP 8.0 or PHP 8.1.
+
+For the environments still running in PHP 7.4 we have released new PHP images which receives backport of security fixes from maintained branches.
```
Comment (wisyhambolu) [inline other-docs/guides/upgrading/v15.md:68]: Change `receives` to `receive`

**Review by wisyhambolu — CHANGES_REQUESTED** [review]
@ferschubert-hm See my suggestions above, minor changes are required.

File: other-docs/guides/upgrading/v15.md (hunk 1 above)
Comment (kovshenin) [inline other-docs/guides/upgrading/v15.md:77]:
```suggestion
Some of our developer focussed documentation has been clarified and improved, taking on board feedback from our customers and partners. 
```

File: other-docs/guides/upgrading/v15.md (hunk 2 above)
Comment (ferschubert-hm) [reply other-docs/guides/upgrading/v15.md:68]: OK, fixed

**Pull request merged**